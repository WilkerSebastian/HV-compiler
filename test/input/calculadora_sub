0-250 140 840 0-50 150 040 350 170 870 606 000