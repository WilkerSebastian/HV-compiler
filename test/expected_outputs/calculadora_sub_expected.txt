250
200
150
100
50
0
